fineract-provider/src/main/resources/sql/migrations/core_db/V265__modify_external_service_schema.sql

```diff
@@ -53,9 +53,9 @@ insert into c_external_service_properties (`name`, `value`, `external_service_id
 
 insert into c_external_service_properties (`name`, `value`, `external_service_id`) values('password', 'support80', (select id from c_external_service where name = 'SMTP_Email_Account'));
 
-insert into c_external_service_properties (`name`, `value`, `external_service_id`) values('host', 'smtp.gmail.com', (select id from c_external_service where name = 'SMTP_Email_Account'));
+insert into c_external_service_properties (`name`, `value`, `external_service_id`) values('host', 'localhost', (select id from c_external_service where name = 'SMTP_Email_Account'));
 
-insert into c_external_service_properties (`name`, `value`, `external_service_id`) values('port', '587', (select id from c_external_service where name = 'SMTP_Email_Account'));
+insert into c_external_service_properties (`name`, `value`, `external_service_id`) values('port', '3025', (select id from c_external_service where name = 'SMTP_Email_Account'));
```

Comment:
Traditionally we tried to avoid changing existing migration files (because then the DB after a migration and a fresh install is not the same anymore) so perhaps you would consider putting this into a new file?

Reply:
The reason why I chose this "unorthodox" approach is that we don't want to overwrite people's existing SMTP configurations. We only want to change the default for new systems that is used in testing. So we don't want to migrate/change any existing setups (as adding a new migration script would do), we want to just change the value that is used when setting up a new system...

Another approach would be to have some custom SQL overwriting the initial values in testing. But as no one should be using the initial values anyway in their systems, why not just change them...

Reply:
SGTM, then we'll do this as an "exception to the rule".